app/routes/base.py

```diff
@@ -34,6 +36,10 @@ class BaseAdminAPIHandler(BaseAPIHandler):
     Base handler for all api/admin/* routes
     """
 
+    async def options(self):
+        self.set_status(204)
+        await self.finish()
```

Comment:
@jaydhulia - Do we want to do any validation here and return a prettier error as opposed to CORS standard error message?

Reply:
`options` is typically not supposed to return any errors. It will just return an OK and the actual route will take care of error handling.